tools/packaging/kata-deploy/kata-deploy/base/kata-deploy-stable.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             fieldRef:
               fieldPath: spec.nodeName
         securityContext:
-          privileged: false
+          privileged: true
```

Comment:
Why `privileged: true`? What is blocking keeping `privileged:  false` ?

Reply:
Accessing /var/run/dbus and /run/systemd/system was not possible with privileged set to false for the Ubuntu based container, IIRC.